fhir/parser.go

```diff
@@ -8,19 +8,17 @@ import (
 
 // ParseCapabilityStatement parses the Capability Statement in the body of the provided http response into a CapabilityStatement struct
 // TODO: Make this function return appropriate version (DSTU2, DSTU3...)
-func ParseCapabilityStatement(resp *http.Response) DSTU2CapabilityStatement {
+func ParseCapabilityStatement(resp *http.Response) (DSTU2CapabilityStatement, error) {
+	var capabilityStatement DSTU2CapabilityStatement
+
 	defer resp.Body.Close()
+
```

Comment:
this is not for this PR - but wouldn't you close `resp.Body` in whatever initially creates it? For example, what if after line 54 in main.go, I want to do something in `resp.Body`? Also, isn't this only called when we're doing `recordLongRunningStats`? And if so, are we leaving `resp.Body` open for all the other rounds of calls?

Reply:
I think it is only "opened" as part of the ReadAll call, so it's not perpetually open regardless.